standard/standard_7.php

```diff
@@ -221,7 +221,7 @@ function unpack (string $format, string $string, int $offset = 0): array|false
 function get_browser (?string $user_agent, bool $return_array): object|array|false
 {}
 
-
+#[PhpStormStubsElementAvailable(to: '7.4')]
```

Comment:
Could you please rollback this change as it's code style related? Overall in stubs attributes are placed between phpdoc and signature

Reply:
Currently, I see the "PhpStormStubsElementAvailable" attributes only over the PHPDoc, and this was the only place where we use it between PHPDoc and signature. Should we move all in between or only this?

Reply:
Thank you for the note! Well, this is rather the issue with other `PhpStormStubsElementAvailable` and if look at other attributes they are placed between phpdoc and signature. So yes, I think it makes sense to move all attributes between phpdoc and signature. Despite all those changes would be style-related but taking into account possible issues like https://youtrack.jetbrains.com/issue/WI-55651 I would move attributes just in case.

Reply:
I will create a new PR for this code style fixes. :+1: